docs/networks/mainnet/parameters.md

```diff
@@ -32,6 +32,16 @@ Developers can use the services of the following 3rd party providers:
 - [NowNodes](https://nownodes.io/) RPC URL: `https://lukso.nownodes.io`
 - [Thirdweb](https://thirdweb.com/) RPC URL: `https://lukso.rpc.thirdweb.com`
 
+We recommend developers to use these RPC providers over our public RPC URL as they provide better performance and stability.
+
+## IPFS Storage
+
+Developers can use the following RPC in order to fetch and store profile or asset data:
+
+- IPFS Download: `https://api.universalprofile.cloud/ipfs`
```

Comment:
We should encourage developers to figure out their own IPFS gateways. We can not support IPFS delivery for the whole community.

Now that our new gateway is propagating files correctly - people can use any IPFS providers (infura pinata etc.) To correctly fetch the assets.

Therefore I recommend to not advertise these URLs here or at the very least to add a warning:

This is for development purposes only. We do not guarantee any SLA and we can add rate limits. You are highly encouraged to get your own IPFS gateway (infura, pinata etc.)